src/Symfony/Component/DependencyInjection/Loader/PhpFileLoader.php

```diff
@@ -108,6 +122,11 @@ class_exists(ContainerConfigurator::class);
         } finally {
             $this->instanceof = [];
             $this->registerAliasesForSinglyImplementedInterfaces();
+
+            if (null !== $this->configBuilderAutoloader) {
```

Comment:
`null` is falsy, a `Closure` is not.
```suggestion
            if ($this->configBuilderAutoloader) {
```